src/Bundle/test/src/AppBundle/Resources/config/doctrine/Author.orm.yml

```diff
@@ -0,0 +1,18 @@
+AppBundle\Entity\Author:
+    type: mappedSuperclass
```

Comment:
```suggestion
    type: entity
```

Reply:
I'd leave it as `mappedSuperclass` as it's a Sylius resource and that's the way it's used in the most cases (so that it allows for extension).